pkg/cli/cmd/bicep/publishextension/publish.go

```diff
@@ -139,23 +134,39 @@ func (r *Runner) Run(ctx context.Context) error {
 	return nil
 }
 
-func generateBicepExtensionIndex(ctx context.Context, inputFilePath string, outputDirectoryPath string) error {
-	// npx @radius-project/manifest-to-bicep-extension@alpha generate <resource provider> <temp>
-	args := []string{
-		"@radius-project/manifest-to-bicep-extension@alpha",
-		"generate",
-		inputFilePath,
-		outputDirectoryPath,
+func generateBicepExtensionIndex(_ context.Context, inputFilePath string, outputDirectoryPath string) error {
+	// Read the radius manifest format
+	radiusManifest, err := manifest.ReadFile(inputFilePath)
+	if err != nil {
+		return clierrors.MessageWithCause(err, "Failed to read manifest file")
 	}
-	cmd := exec.CommandContext(ctx, "npx", args...)
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
-
-	err := cmd.Run()
+	
+	// Convert to bicep-tools-go format
+	bicepManifest, err := convertRadiusToBicepManifest(radiusManifest)
+	if err != nil {
+		return clierrors.MessageWithCause(err, "Failed to convert manifest format")
+	}
+	
+	generator := cli.NewGenerator()
```

Comment:
Why is this reference a CLI that we reference as a library? Or maybe this is a temporary test.